packages/x-components/src/x-modules/semantic-queries/components/semantic-queries.vue

```diff
@@ -1,14 +1,34 @@
 <template>
-  <NoElement v-if="$x.semanticQueries.length">
-    <slot name="default" :queries="queries">
+  <NoElement v-if="semanticQueries.length">
+    <!-- eslint-disable max-len -->
+    <!--
+      @slot Semantic Query content
+        @binding {string[]} queries - SemanticQueries mapped to strings
+        @binding {SemanticQuery[]} semanticQueries - SemanticQueries
+        @binding {(query: string) => SemanticQuery} findSemanticQuery - Method to find a semantic query given a query
+    -->
+    <!-- eslint-enable max-len -->
+    <slot name="default" v-bind="{ queries, semanticQueries, findSemanticQuery }">
       <ul class="x-semantic-queries">
```

Comment:
Check if we can do this implementation with `BaseSuggestions` instead

Reply:
Done 👌